i18n/i18n.go

```diff
@@ -56,14 +57,18 @@ func (i18n *I18n) AddTransaltion(translation *Translation) {
 	i18n.Translations[translation.Locale][translation.Key] = translation
 }
 
-func (i18n *I18n) SaveTransaltion(translation *Translation) {
+func (i18n *I18n) SaveTranslation(translation *Translation) (err error) {
```

Comment:
that change is included in this PR too...
https://github.com/qor/qor/pull/15/files#r34331896